crypto/fipsmodule/sha/keccak1600.c

```diff
@@ -349,7 +351,7 @@ void KeccakF1600(uint64_t A[KECCAK1600_ROWS][KECCAK1600_ROWS]) {
     // Neoverse V1 and V2 do support SHA3 instructions, but they are only
     // implemented on 1/4 of Neon units, and are thus slower than a scalar
     // implementation.
-
+#if defined(OPENSSL_AARCH64)
```

Comment:
Nit: Maybe you can && this line and the one below to match l.429.
When you add the brackets as per Dusan's comment, you can maybe do this small change, if you want.

Reply:
Actually, these are not ending at the same place. There is one additional _non_-`s2n-bignum` assembly implementation for arm, so it should fall back to it if not `s2n-bignum` but still `aarch64` is detected.